PencilCase/PencilCase/scripts/application.ts

```diff
@@ -18,6 +20,68 @@ function onDeviceReady(): void {
     //var receivedElement = parentElement.querySelector('.received');
     //listeningElement.setAttribute('style', 'display:none;');
     //receivedElement.setAttribute('style', 'display:block;');
+
+    initializeDatabase();
+}
+
+function initializeDatabase() {    
+    let db = window.openDatabase("PencilCase", "0.1", "Pencil Case", 2 * 1024 * 1024);
+    if (db) {
+        db.transaction((transaction) => {
+            // Uncomment the line below if need to re-create the table, like adding/removing/changing columns
+            //transaction.executeSql('DROP TABLE IF EXISTS Product', [], null, onDBError);
+            transaction.executeSql('CREATE TABLE IF NOT EXISTS Product (\
```

Comment:
SqlLite is not using standard SQL. For more information, refer to the comments of this story for more details, or refer to your one note.